.github/workflows/deploy.yml

```diff
@@ -11,6 +11,7 @@ on:
       - 2024-07
       - 2024-10
       - 20[0-9][0-9]-[01][1470]
+      - 20[0-9][0-9]-[01][1470]-rc
```

Comment:
I don't think we should add to this because it's for stable releases. Shouldn't we have a new workflow that does the prerelease steps? Ideally we wouldn't need a Version PR to ship these prereleases

Reply:
> Ideally we wouldn't need a Version PR to ship these prereleases

I'm not sure how else this would work? I _think_ only snapshot releases support this. But I'm happy to be proven wrong

Reply:
Ok, I'm good with that. I think the Version PRs can only be approved by certain ppl though. You need to add a new protection rule to bypass that